src/opnsense/scripts/OPNsense/CaptivePortal/htdocs_default/index.html

```diff
@@ -173,7 +173,7 @@
               <form class="form-signin">
                   <h2 class="form-signin-heading">Please sign in</h2>
                   <label for="inputUsername" class="sr-only">Username</label>
-                  <input type="text" id="inputUsername" class="form-control" placeholder="Username" required autofocus>
+                  <input type="text" id="inputUsername" class="form-control" placeholder="Username" required autofocus autocomplete="none" autocapitalize="none" autocorrect="off">
```

Comment:
`autocapitalize`
Specifies the auto-capitalization behavior of text elements. Possible values are:

`none`
Do not capitalize any text automatically.
Available in iOS 5.0 and later.

`autocorrect`
Specifies a form field allows automatic correction. If `off`, deactivates automatic correction for a form or input element; `on`, activates automatic correction for the element.

Support Level
Apple extension.

`autocomplete`
Specifies a form field accepts AutoFill information. If `off`, deactivates automatic completion for an input element; `on`, activates automatic completion for the element.

Related Tags
`input`